aries-site/src/pages/templates/filtering.mdx

```diff
@@ -5,18 +5,19 @@ import {
   FilteringWithRangeSelector,
   FilteringWithDropButton,
   FilteringWithSelect,
-  PersistentFiltering,
 } from '../../examples';
 
+### Filtering with DropButton
+
 <Example
-  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/PersistentFiltering.js"
-  figma="https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=681%3A1430"
-  showResponsiveControls={['laptop', 'mobile']}
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/filtering/FilteringWithDropButton.js"
   template
+  showResponsiveControls={['laptop', 'mobile']}
 >
-  <PersistentFiltering />
+  <FilteringWithDropButton />
```

Comment:
Proposing we move the "Layer as right side panel" one to the top instead. it demonstrates a bit richer of filtering experience. If we do, we don't need to bring the heading with it, just rename the JSX here and the code link